manifests/mod/alias.pp

```diff
@@ -6,6 +6,7 @@
     'redhat'  => '/var/www/icons',
     'freebsd' => '/usr/local/www/apache22/icons',
   }
+  $icons_index = 'Indexes Multiviews'
```

Comment:
I think this should be added as a parameter for `apache::mod::alias` if you actually want to be able to set it.